specification/ai/data-plane/Face/preview/v1.2-preview.1/Face.json

```diff
@@ -0,0 +1,9272 @@
+            "enum": [
+              "recognition_01",
+              "recognition_02",
+              "recognition_03",
+              "recognition_04"
+            ],
+            "x-ms-enum": {
+              "name": "RecognitionModel",
+              "modelAsString": true,
+              "values": [
+                {
+                  "name": "recognition_01",
+                  "value": "recognition_01",
+                  "description": "The default recognition model for \"Detect\". All those faceIds created before 2019 March are bonded with this recognition model."
+                },
+                {
+                  "name": "recognition_02",
+                  "value": "recognition_02",
+                  "description": "Recognition model released in 2019 March."
+                },
+                {
+                  "name": "recognition_03",
+                  "value": "recognition_03",
+                  "description": "Recognition model released in 2020 May."
+                },
+                {
+                  "name": "recognition_04",
+                  "value": "recognition_04",
+                  "description": "Recognition model released in 2021 February. It's recommended to use this recognition model for better recognition accuracy."
+                }
+              ]
+            }
+          },
+          {
+            "name": "persistedFaceId",
+            "in": "path",
+            "description": "Face ID of the face.",
+            "required": true,
+            "type": "string",
+            "format": "uuid"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "A successful call returns an empty response body. The service has accepted the request and will start processing soon. The client can query the operation status and result using the URL specified in the 'Operation-Location' response header. The URL expires in 48 hours.",
+            "headers": {
+              "operation-Location": {
+                "type": "string",
+                "format": "uri",
+                "description": "The location of an instance of OperationResult"
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/FaceErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "String error code indicating what went wrong."
+              }
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/session/sessionImages/{sessionImageId}": {
+      "get": {
+        "operationId": "LivenessSessionOperations_GetSessionImage",
+        "description": "Get session image stored during the liveness session.",
+        "produces": [
+          "application/octet-stream",
+          "application/json"
```

Comment:
I believe it only produces "application/octet-stream". This line should be removed

Reply:
It's for error response, 4xx/5xx response will still be json.